x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/types.ts

```diff
@@ -69,6 +69,7 @@ interface EmailConfig {
   host: string;
   port: number;
   secure?: boolean;
+  hasAuth?: boolean;
```

Comment:
Will the `hasAuth` always have a value with the migration in place?

```suggestion
  hasAuth: boolean;
```